ODIN_II/SRC/include/odin_error.h

```diff
@@ -28,6 +28,8 @@ enum odin_error {
     NETLIST,
     /* for blif parser errors */
     PARSE_BLIF,
+    /* for blif output errors */
+    OUTPUT_BLIF,
     /* for errors in the netlist simulation */
```

Comment:
probably somewhere in the code pointed to the last element of odin_error enum instead of using a related name.

Reply:
I think this could be the reason? https://github.com/CAS-Atlantic/vtr-verilog-to-routing/blob/976a83f25447cac3b630103c51fd7ac2d1ebd2cf/ODIN_II/SRC/odin_error.cpp#L108

Reply:
this is the cause of failures, have you fixed it?